hoodie-client/src/test/java/com/uber/hoodie/TestHoodieClient.java

```diff
@@ -158,7 +157,7 @@ public void testFilterExist() throws Exception {
 
         JavaRDD<HoodieRecord> smallRecordsRDD = jsc.parallelize(records.subList(0, 75), 1);
         // We create three parquet file, each having one record. (two different partitions)
-        List<WriteStatus> statuses = writeClient.bulkInsert(smallRecordsRDD, newCommitTime).collect();
+        List<WriteStatus> statuses = writeClient.bulkInsert(smallRecordsRDD, newCommitTime, Option.empty()).collect();
```

Comment:
lets provide an overloaded method `bulkInsert(rdd, commitTime)` in the HoodieWriteClient itself, so its consistent with other APIs.